src/backend/distributed/commands/sequence.c

```diff
@@ -317,6 +317,34 @@ PreprocessDropSequenceStmt(Node *node, const char *queryString,
 }
 
 
+/*
+ * SequenceDropStmtObjectAddress returns list of object addresses in the drop sequence
+ * statement.
+ */
+List *
+SequenceDropStmtObjectAddress(Node *stmt, bool missing_ok)
+{
+	DropStmt *dropSeqStmt = castNode(DropStmt, stmt);
+
+	List *objectAddresses = NIL;
+
+	List *deletingSequencesList = dropSeqStmt->objects;
+	List *objectNameList = NULL;
+	foreach_ptr(objectNameList, deletingSequencesList)
+	{
+		RangeVar *seq = makeRangeVarFromNameList(objectNameList);
+
+		Oid seqOid = RangeVarGetRelid(seq, NoLock, missing_ok);
```

Comment:
same concern here: Does passing `missing_ok` directly here satisfy our needs for the check-vanilla? If so, I'm fine with that

Reply:
Also, why do we have `NoLock` here. We should at least get AccessShareLock

Reply:
missing_ok is safe to use, explained above. I can actually acquire an AccessShareLock.